docs/design/fabric-v2+/fpc-cc-invocation.puml

```diff
@@ -127,14 +133,22 @@ group invoke
       end ref
 
 
-      Enclave1 -> Enclave1 : result_enc <- encrypt result with r_key
+      Enclave1 -> Enclave1 : enc_ccrsp_msg.encrypted_response <- encrypt result with r_key
       note right
         The encryption is via authenticated symmetric cipher like aes-gcm
       end note
-      Enclave1 -> Enclave1 : Sig_Enclave <- sign <args, read/writeset, result_enc> with Enclave_SK
-      return <result_enc, Sig_Enclave>
+      Enclave1 -> Enclave1 : enc_ccrsp_msg.signature <- sign <args, read/writeset, result_enc> with Enclave_SK
+      return <enc_ccrsp_msg>
+
+    alt lightweight FPC
+      enc_ccrsp_msg.encrypted_rw_set <- encrypted_rw_set
```

Comment:
I would just call it `rw_set`. Values of the rw set are may be encrypted but not the entire rws_set. In contrast, encrypted_response is a ciphertext of a response msg. for the rw_set I think this is not the case.

Reply:
ok, i guess the comment in fpc.proto that it is rw set of encrypted values i guess should be enough; i guess i can add also a comment here ...